lib/dartino/device/device.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:async';
+
+import 'package:atom_dartlang/atom.dart';
+
+import '../dartino_util.dart';
+import '../launch_dartino.dart';
+import '../sdk/dartino_sdk.dart';
+import 'stm32f746disco.dart';
```

Comment:
Point taken, but these are the names used in the dartino project, and I want to keep them the same in both atom and in dartino for my own sanity.